### ISSUE TITLE
请问有自带中国人声音的speaker吗

### ISSUE THREAD

**Issue by richardjian2023** [netease-youdao/EmotiVoice#48]
看了一下speaker列表似乎都是外国人？有中国人的speaker吗，还是说需要自己在哪里下载导入到什么地方，谢谢啦

**Comment by syq163** [comment]
Yes, currently all the speakers are foreigners. We are also planning to include more Chinese speakers.
Looking forward to your Recommendations at [Need Recommendations and contributions of good Voices (In progress)](https://github.com/netease-youdao/EmotiVoice/wiki/Need-Recommendations-and-contributions-of-good-Voices)

**Comment by song4875343** [comment]
你的示例中，显然有中国人的，你没放出来而已，这个和barker有啥区别，好像区别不大啊

**Comment by lucasjinreal** [comment]
为何不放出来呢？中国人的语音不好吗?

**Comment by friendmine** [comment]
我觉得啊，一个优秀的开源系统是给你一个基础，然后有足够的自定义能力，你不能要求一个跟商业版本完全一致的功能吧。
毕竟开源这类东西，涉及到数据版权、隐私一类的问题。

**Comment by lucasjinreal** [comment]
隐私跟 有但是不放出来是两码事，中国语音开放出来是为了推动中文的发展，如果有中文却故意不放出来，那还不如不开源。

**Comment by jzt-Tesla** [comment]
是真的坏而且蠢，明明是中国人开发的软件，只去搞洋人的语音你就高大上了？你这种软件外国人根本不会去使用，外国比你们强大的多的软件一大堆，所有呢？只是为了完成KPI，现在好了？估计团队都毕业了吧，哈哈 ~
